code/src/OSFramework/Enum/ErrorCodes.ts

```diff
@@ -69,6 +69,7 @@ namespace OSFramework.Enum {
         API_FailedFreezeColumnsByActiveCell = 'GRID-API-10003',
         API_FailedHasFrozenColumns = 'GRID-API-10004',
         API_FailedUnfreezeColumns = 'GRID-API-10005',
+        API_FailedAddColumnToGroupPanel = 'GRID-API-10006',
```

Comment:
I think this might be a conflict with other PR: https://github.com/OutSystems/outsystems-datagrid-reactive/pull/239/files

Reply:
I'll use the GRID-API-10007